app/presenters/summary/sections/more_information.rb

```diff
@@ -1,10 +1,6 @@
 module Summary
   module Sections
     class MoreInformation < Sections::BaseSection
-      def show?
-        FeatureFlags.more_information.enabled?
-      end
```

Comment:
Removed as I believe this will always display? Not sure however so correct me if wrong!

Reply:
Yep. Section always shown, and content defaulted to 'None' if nothing entered